.gitignore

```diff
@@ -7,6 +7,7 @@
 *.dylib
 bin
```

Comment:
now, we have files to be committed into bin folder. 

Do we need to change bin from a folder to specific files like
```
bin/support-bundle
bin/preflight
```

Reply:
Perhaps `!bin/watchrsync.js` would be better. Ignore all files in the `bin` directory except `bin/watchrsync.js`

Reply:
> I think we have npm-install set gaze-run-interrupt into 2.0.0, so should be ok without package.json.

I also added a make job that installs the right npm packages

Reply:
> Perhaps `!bin/watchrsync.js` would be better. Ignore all files in the `bin` directory except `bin/watchrsync.js`

I don't think it's necesary; I checked with a fresh checkout of my branch and it works as-is:

```
 ✔  alice  /tmp/troubleshoot   watchrsync →ada/watchrsync  cd bin
 ✔  alice  /tmp/troubleshoot/bin   watchrsync →ada/watchrsync  ls
watchrsync.js
 ✔  alice  /tmp/troubleshoot/bin   watchrsync →ada/watchrsync  touch foo
 ✔  alice  /tmp/troubleshoot/bin   watchrsync →ada/watchrsync  touch foobar
 ✔  alice  /tmp/troubleshoot/bin   watchrsync →ada/watchrsync  mkdir shoop
 ✔  alice  /tmp/troubleshoot/bin   watchrsync →ada/watchrsync  touch shoop/dawoop
 ✔  alice  /tmp/troubleshoot/bin   watchrsync →ada/watchrsync  ls
foo           foobar        shoop         watchrsync.js
 ✔  alice  /tmp/troubleshoot/bin   watchrsync →ada/watchrsync  gst
On branch watchrsync
Your branch is up to date with 'origin/ada/watchrsync'.

nothing to commit, working tree clean
```